apps/web/app/(landing)/login/LoginForm.tsx

```diff
@@ -45,6 +67,84 @@ export function LoginForm() {
     setLoadingMicrosoft(false);
   };
 
+  const handleSSOSignIn = () => {
+    setShowSSOForm(true);
+  };
+
+  const onSubmitSSO: SubmitHandler<SSOFormData> = async (data) => {
+    try {
+      const url = new URL("/api/sso/signin", window.location.origin);
+      url.searchParams.set("email", data.email);
+      url.searchParams.set("organizationSlug", data.organizationName);
+
+      const response = await fetch(url.toString());
+      const responseData = await response.json();
+
+      if (!response.ok) {
+        toastError({
+          title: "SSO Sign-in Error",
+          description: responseData.error || "Failed to initiate SSO sign-in",
+        });
+        return;
+      }
+
+      if (responseData.redirectUrl) {
+        toastSuccess({ description: "Redirecting to SSO provider..." });
+        window.location.href = responseData.redirectUrl;
+      }
+    } catch {
+      toastError({
+        title: "SSO Sign-in Error",
+        description: "An unexpected error occurred. Please try again.",
+      });
+    }
+  };
+
+  const handleBackToSocial = () => {
+    setShowSSOForm(false);
+    reset();
+  };
+
+  if (showSSOForm) {
```

Comment:
Just make this a new page. We don't need to reinvent the back button.

`/login/sso`